solc/CommandLineInterface.cpp

```diff
@@ -905,8 +905,10 @@ void CommandLineInterface::handleCombinedJSON()
 		if (requests.count("metadata"))
 			contractData["metadata"] = m_compiler->metadata(contractName);
 		if (requests.count(g_strBinary))
+			// Do we want to change it here or not?
```

Comment:
What do you mean with this comment? Is this a left-over?

Reply:
It is a question to the reviewers.

Reply:
Well, I wasn't sure :) I'd prefer to add such questions as regular comments to the pull request. In that case, I'd say yes, let's be consistent and also add the reference hints to the ``combined-json`` output.

Reply:
Ok, perhaps I should give the reason why I did not add it here: I would assume some tools to be confused when a json string suddenly starts containing newlines, and several tools might not be prepared for that.